crates/sui-open-rpc/samples/transactions.json

```diff
@@ -452,30 +419,30 @@
           "dataType": "moveObject",
           "type": "0x2::Coin::Coin<0x2::SUI::SUI>",
           "fields": {
-            "balance": 98957,
+            "balance": 98935,
             "id": {
-              "id": "0x0a3c453b666a137440cd297ae73cebfeee7a40b3",
+              "id": "0x15a8506c8549848bad388e0597b0c147860e23db",
               "version": 3
             }
           }
         },
         "owner": {
-          "AddressOwner": "0x51372a6dcf5799e4c39f7cd6bc68e27bdc92728c"
+          "AddressOwner": "0x08177f0d3cb878f43ed6f6d1990629b0ce3d3122"
         },
-        "previousTransaction": "SLiGKy49vDHS+00YR2ue3iZ2sLzzmRqm//ktNxQAUa0=",
+        "previousTransaction": "T3bPBfOtXqJCB4JvqdXcqP7l+v3NApVimV+k+cHlIks=",
         "storageRebate": 15,
         "reference": {
-          "objectId": "0x0a3c453b666a137440cd297ae73cebfeee7a40b3",
+          "objectId": "0x15a8506c8549848bad388e0597b0c147860e23db",
           "version": 3,
-          "digest": "gjB25CxYgdZ6NKlZU0Y2dJBjtjAQavHtE/ViJKuBJh4="
+          "digest": "ooX070rATxGaOVSdFedfkQWoVGKjjcbrnGyCZq2XFAA="
         }
       }
     }
   },
   "publish": {
```

Comment:
I noticed that this does not include a publish event as defined in `SuiEvent`. Is this expected?

Reply:
yes it's expected, the publish event is a derived event (not from the vm) and it only contain the package id, so not very useful to be included in the publish response which have those data already.